src/net.cpp

```diff
@@ -2492,6 +2532,14 @@ void CConnman::StopNodes()
     if (fAddressesInitialized) {
         DumpAddresses();
         fAddressesInitialized = false;
+
+        auto anchors_to_dump = GetCurrentBlockRelayOnlyConns();
```

Comment:
I suggest you add a brief comment here to say that anchor connections are only dumped during clean shutdown.

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/17428#issuecomment-693330527).